cmd/object-lock.go

```diff
@@ -63,6 +65,8 @@ var (
 	errUnknownWORMModeDirective    = errors.New("unknown WORM mode directive")
 	errObjectLockMissingContentMD5 = errors.New("Content-MD5 HTTP header is required for Put Object requests with Object Lock parameters")
 	errObjectLockInvalidHeaders    = errors.New("x-amz-object-lock-retain-until-date and x-amz-object-lock-mode must both be supplied")
+
+	ntpServer = env.Get("MINIO_NTP_SERVER", "0.us.pool.ntp.org")
```

Comment:
Can this be turned off?

Reply:
It can't be turned off @kannappanr

Reply:
Why not?

Reply:
This is meant for retention alone @klauspost it is assumed that we can't rely on system clock